core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -140,11 +146,14 @@ protected FileInStream(URIStatus status, InStreamOptions options, FileSystemCont
     mShouldCache = mAlluxioStorageType.isStore();
     mShouldCachePartiallyReadBlock = options.isCachePartiallyReadBlock();
     mClosed = false;
-    mLocationPolicy = options.getLocationPolicy();
+    mCacheLocationPolicy = options.getLocationPolicy();
```

Comment:
I think we can keep `getLocationPolicy` for backward compatibility but deprecate it and use `getCacheLocationPolicy` instead? If you anticipate consolidating policies soon, I think it is fine to leave it as is.